lib/types/proposals.ts

```diff
@@ -1,9 +1,14 @@
-import { MergeWithOptionalFields } from "../types";
-import { IdlAccounts } from "@coral-xyz/anchor";
-import { AutocratV0 } from "../idl/autocrat_v0";
-import { AutocratV0 as AutocratV0_1 } from "../idl/autocrat_v0.1";
-import { ProposalAccountWithKey } from "../types";
-import { VaultAccount } from "./conditionalVault";
+import { AccountWithKey, MergeWithOptionalFields } from "../types";
+import { IdlAccounts, IdlTypes } from "@coral-xyz/anchor";
+import { AutocratV0 } from "@/idl/autocrat_v0";
+import { AutocratV0 as AutocratV0_1 } from "@/idl/autocrat_v0.1";
+import { VaultAccount } from "@/types";
+
+export type ProposalInstruction = MergeWithOptionalFields<
```

Comment:
We need a solution for this, https://github.com/metaDAOproject/futarchy-ts/issues/17